api/server.js

```diff
@@ -21,20 +21,20 @@ manifestPromise.then(manifest => {
     server.auth.strategy('email_token', 'email_token', {
       findToken: async id => {
         try {
```

Comment:
You should just remove the `try { } catch ()` completely. The result is that any error thrown will be thrown up the chain without having the extra lines